content/300-guides/100-prisma-guides/100-prisma-migrate-guides/100-add-prisma-migrate-to-a-project.mdx

```diff
@@ -0,0 +1,65 @@
+---
+title: Add Prisma Migrate to an existing project
+---
+
+## Prerequisites and assumptions
+
+This guide relies on the following prerequisites:
+
+* You have added Prisma to your project
+* 
+
+Be aware that:
+
+* You cannot combine your current migration history with Prisma Migrate
+
+## Upgrade development environments
+
+1. Make sure you Prisma schema is in sync with your database schema. To make sure, introspect your database:
+
+   ```terminal
+   npx prisma introspect
```

Comment:
Do we need to add a small section about install @prisma/cli first?
Because running `npx prisma` will load prisma1